universe/Species.h

```diff
@@ -105,7 +105,7 @@ class FO_COMMON_API Species {
 
     std::string                     Dump(unsigned short ntabs = 0) const;                                   ///< returns a data file format representation of this object
     const std::vector<FocusType>&   Foci() const                        { return m_foci; }                  ///< returns the focus types this species can use
-    const std::string&              DefaultFocus() const                { return m_default_focus; }       ///< returns the name of the planetary focus this species prefers. Default for new colonies and may affect happiness if on a different focus?
+    const std::string&              DefaultFocus() const                { return m_default_focus; }       ///< returns the name of the planetary focus this species prefers. Default for new colonies and will affect happiness if on a different focus
```

Comment:
Default focus can be different from liked focus, no need to talk about happiness/stability here. How about 
"//< returns the name of the planetary focus this species defaults to. Used for new colonies and uninvaded natives."

Reply:
the comment was there, long before i saw it. i only changed it from a question to a statement which i believed to be correct at the time. i can change the comment back to what it was before thought?

Reply:
whatever. i'll copy/paste your comment.

Reply:
thx. the original comment was also not completely valid anymore. small typo in your change though

Reply:
i'll correct that.